docs/UserGuide.md

```diff
@@ -95,6 +95,21 @@ ModuleMate Finder is a desktop app that allows contacts to find people taking th
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* Below is a table depicting the parameters used in Module Mate Finder and their respective constraints
+
+| Parameter | Constraint                                                                                 | Example             |
+|-----------|--------------------------------------------------------------------------------------------|---------------------|
+| Index     | Index must be a positive integer and be less than Integer.MAX_VALUE                        | 1, 2, 3             |
```

Comment:
Kinda of CS2101 stuff, but Integer.MAX_VALUE might not be understandable for target audience. Maybe `2147483647` or `2^31 - 1`? Maybe can think of better alternative?